tests/ert/unit_tests/gui/tools/plot/test_plot_api.py

```diff
@@ -329,6 +330,41 @@ def test_plot_api_handles_non_existant_gen_kw(api_and_storage):
     assert api.data_for_parameter(str(ensemble.id), "gen_kw:does_not_exist").empty
 
 
+def test_plot_api_handles_colons_in_parameter_keys(api_and_storage):
+    api, storage = api_and_storage
+    experiment = storage.create_experiment(
+        parameters=[
+            GenKwConfig(
+                name="group",
+                forward_init=False,
+                update=False,
+                transform_function_definitions=[
+                    TransformFunctionDefinition(
+                        name="subgroup:1:2:2", param_name="NORMAL", values=[0, 0.1]
```

Comment:
maybe using `RAW` distribution would be enough and then the expected value will be the same as stored.